tests/unit/api.py

```diff
@@ -1204,6 +1204,21 @@ def test_list_projects(self, db):
             'runway',
         ]
 
+    import uuid
+
+    def test_list_networks(self, db):
+        assert json.loads(api.list_networks()) == {}
+        api.project_create('anvil-nextgen')
+        network_create_simple('netA', 'anvil-nextgen')
```

Comment:
Reading docs/networks.md and docs/rest-api.md should give you the info you need on what the different network creation arguments mean.